### DISCUSSION TITLE
I want to connect to Miningcore ETHERUM pool by renting hashpower from Miningrigrentals.

### DISCUSSION THREAD

**Discussion by konstiantyn** [coinfoundry/miningcore#1108] [Q&A]
I want to connect to Miningcore ETHERUM pool by renting hashpower from Miningrigrentals.
When I try to connect to miningcore I get this alert.
"We were unable to establish a connection to the pool you provided."

**Comment by konstiantyn** [comment]
It works.

**Reply by oliverw** [reply]
So what, does it work or does it not?

**Comment by konstiantyn** [comment]
working well.Thanks

**Reply by xsiroxcoin** [reply]
what setting you do i same issue face
